src/MongoDB/Driver/Manager.cpp

```diff
@@ -563,6 +728,7 @@ void HHVM_METHOD(MongoDBDriverManager, __construct, const String &dsn, const Arr
 	data->m_client = client;
 
 	hippo_mongo_driver_manager_apply_ssl_opts(data->m_client, driverOptions);
+	hippo_mongo_driver_manager_set_monitoring_callbacks(data->m_client, this_);
```

Comment:
We discussed that before, while discussing the removal of the callbacks at the end of each request; and decided this was the safer thing to do.